core/api/src/main/java/com/microsoft/gctoolkit/event/zgc/ZGCConcurrentPhases.java

```diff
@@ -2,7 +2,10 @@
 // Licensed under the MIT License.
 package com.microsoft.gctoolkit.event.zgc;
 
-public enum ZGCConcurrentPhases {
+import com.microsoft.gctoolkit.event.GarbageCollectionTypes;
+import com.microsoft.gctoolkit.event.LabelledGCEventType;
+
+public enum ZGCConcurrentPhases implements LabelledGCEventType {
```

Comment:
Just so you know, ZGC and Shenandoah are not fully baked.

Reply:
Indeed, am I'm also getting an exception when running the sample. Hopefully this can all be finished soon.